### PR TITLE
Particles Lab

### PR DESCRIPTION
**Pull request by ghost**
Markdown file for instructions, added portion related to Maya lab option.

### CODE REVIEW

File: lab08/README.md [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+
+Check out some of these examples:
+
+-   [https://threejs.org/examples/?q=particle\#webgl_gpu_particle_system](https://www.google.com/url?q=https://threejs.org/examples/?q%3Dparticle%23webgl_gpu_particle_system&sa=D&source=editors&ust=1630507972544000&usg=AOvVaw3DAjD1MyqdJAQl63y7wkwN)
+-   [https://lab.hakim.se/trail/02/](https://www.google.com/url?q=https://lab.hakim.se/trail/02/&sa=D&source=editors&ust=1630507972545000&usg=AOvVaw1m7LRCnC77WvwWc8F2gKa-)
+-   [https://www.youtube.com/watch?v=CMVRILPTWzU](https://www.google.com/url?q=https://www.youtube.com/watch?v%3DCMVRILPTWzU&sa=D&source=editors&ust=1630507972545000&usg=AOvVaw2okLwp0HuENJ8ihs-tMF11)
+
+# Particles: Maya Version
+
+In lab (??) we explored how to use Maya. Maya is useful for creating
```
Comment (dritchie) [inline lab08/README.md]: Can you fill in these (??) sections?

**Review by dritchie — COMMENTED** [review]
See the comment I left on README.md

File: lab08/README.md (hunk 1 above)
Comment (ghost) [reply lab08/README.md]: done!

**Review by ctrotz — APPROVED** [review]
Looks great!

**Pull request merged**